modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/ObjectFieldFormBase.tsx

```diff
@@ -437,7 +437,7 @@ export default function ObjectFieldFormBase({
 				{values.businessType === 'Picklist' &&
 					validListTypeDefinitionId && (
 						<ClayToggle
-							disabled={disabled}
+							disabled={true}
```

Comment:
Hi @diogomelo09, if you simply set the disable property as true, the user will never be able to mark any picklist fields as state. The issue described in the ticket is just related with system objects. So you should set the property as `disabled={disabled || objectDefinition?.system}`.

Reply:
Just did it!! :peepo-clap: